public/fixtures/iiif/collection/sample-sites.json

```diff
@@ -0,0 +1,180 @@
+          "label": {
+            "en": [
+              "The Botanical photography of Alan S. Heilman"
+            ]
+          },
+          "type": "Text",
+          "format": "text\/html"
+        }
+      ]
+    },
+    {
+      "id": "https://digital.lib.utk.edu/assemble/collection/collections/rftacuratedart",
+      "type": "Collection",
+      "label": {
+        "none": [
+          "The Chimney Tops 2 Wildfires In Memory And Art"
+        ]
+      },
+      "thumbnail": [
+        {
+          "id": "https://digital.lib.utk.edu/iiif/2/collections~islandora~object~rftaart%3A6~datastream~JP2/full/300,224/0/default.jpg",
+          "height": 224,
+          "width": 300,
+          "type": "Image",
+          "format": "image\/jpeg"
+        }
+      ],
+      "homepage": [
+        {
+          "id": "https://canopy-iiif-git-rfta-artists-mathewjordan.vercel.app/",
+          "label": {
+            "en": [
+              "The Chimney Tops 2 Wildfires In Memory And Art"
+            ]
+          },
+          "type": "Text",
+          "format": "text\/html"
+        }
+      ]
+    },
+    {
+      "id": "https://iiif.bodleian.ox.ac.uk/iiif/collection/canonici",
+      "type": "Collection",
+      "label": {
+        "none": [
+          "Canonici"
+        ]
+      },
+      "thumbnail": [
+        {
+          "id": "https://iiif.bodleian.ox.ac.uk/iiif/image/fa2bfd65-226a-4794-a444-ea89ee3fc3cf/full/256,/0/default.jpg",
+          "height": 258,
+          "width": 171,
+          "type": "Image",
+          "format": "image\/jpeg"
+        }
+      ],
+      "homepage": [
+        {
+          "id": "https://canopy-iiif-git-canonici-mathewjordan.vercel.app/",
+          "label": {
+            "en": [
+              "Canonici"
+            ]
+          },
+          "type": "Text",
+          "format": "text\/html"
+        }
+      ]
+    },
+    {
+      "id": "https://digital.lib.utk.edu/assemble/collection/gsmrc/kintner",
+      "type": "Collection",
+      "label": {
+        "none": [
+          "Panoramic Images of Elgin P. Kintner"
+        ]
+      },
+      "thumbnail": [
+        {
+          "id": "https://digital.lib.utk.edu/iiif/2/collections~islandora~object~kintner%3A4~datastream~JP2/full/525,150/0/default.jpg",
+          "height": 150,
+          "width": 525,
+          "type": "Image",
+          "format": "image\/jpeg"
+        }
+      ],
+      "homepage": [
+        {
+          "id": "https://canopy-iiif-git-kintnerexamplesite-mathewjordan.vercel.app",
+          "label": {
+            "en": [
+              "Panoramic Images of Elgin P. Kintner"
+            ]
+          },
+          "type": "Text",
+          "format": "text\/html"
+        }
+      ]
+    }
+  ],
+  "homepage": [
+    {
+      "id": "https://canopy-iiif.vercel.app/",
+      "label": {
+        "en": [
+          "Canopy IIIF Collection"
+        ]
+      },
+      "type": "Text",
+      "format": "text\/html"
+    }
+  ]
```

Comment:
Do we need homepage here? The **View All** button on the Bloom component will render if it exists, but I'm not sure if that's necessary in this implementation.

```suggestion
```

![image](https://user-images.githubusercontent.com/7376450/218474466-7cdec833-b159-41d0-b673-72e765c31f07.png)